packages/react-toast/src/components/index.tsx

```diff
@@ -0,0 +1,34 @@
+export { default as Toast } from "./Toast";
+import React from "react";
+import styled from "styled-components";
+
+import Body from "./Body";
+import { toastStyles } from "./Toast/styled";
+import { ICourierToastMessage } from "./Toast/types";
+
+const Styled = styled.div(toastStyles);
+
+export const ToastBody: React.FunctionComponent<ICourierToastMessage> = (
+  props
+) => {
+  return (
+    <Styled className="Toastify__toast-container">
+      <div className="Toastify__toast Toastify__toast--default">
+        <div className="Toastify__toast-body">
+          <Body {...props} />
+        </div>
+        <div
+          className="Toastify__progress-bar Toastify__progress-bar--animated Toastify__progress-bar--default"
+          style={{
+            animation: "none",
+
+            // will want to tweak these after we get them inside studio
```

Comment:
do we need to keep the commented block?

Reply:
i wanna tweak these settings as i play with it in studio